library/hawkular_alerts_group_trigger.py

```diff
@@ -184,23 +204,29 @@ def conditions_update_required(self, trigger_id, desired_conditions):
                     return True
         return False
 
-    def required_updates(self, trigger, name, severity, enabled):
+    def required_updates(self, trigger, name, event_text, severity, auto_resolve, tags, enabled):
         """ Checks whether an update is required for the group trigger
 
             Returns:
-                Empty Hash (None) - If the name, severity and enabled status passed
+                Empty Hash (None) - If the name, event_text, severity, auto_resolve, tags and enabled status passed
                                     equals the group trigger's current values
-                Hash of Changes   - Changes that need to be made if the name, severity
-                                    or enabled status are different than the current
+                Hash of Changes   - Changes that need to be made if the name, event_text, severity, auto_resolve,
+                                    tags or enabled status are different than the current
                                     values of the group trigger.
         """
         updates = {}
 
         # `is not None` check verifies that omitted module params will not be updated
         if (name is not None and trigger.name != name):
             updates["name"] = name
+        if (event_text is not None and trigger.event_text != event_text):
```

Comment:
As this grows (and will continue to) I think we should pass a dictionary of fields and iterate over them. something like:
```python
for attribute in group_trigger_attributes:
    if (attribute is not None and getattr(trigger, attribute) != attribute):
        updates[attribute] = attribute

```
@cben what do you think?